matlab/src/matlab/+arrow/+type/Type.m

```diff
@@ -41,6 +42,29 @@
         function typeID = get.ID(obj)
             typeID = arrow.type.ID(obj.Proxy.getTypeID());
         end
+
+        function F = field(obj, idx)
```

Comment:
`getField` may be better.

Reply:
That's a good point. However, if we renamed `field` to `getField`, it would be inconsistent with other method names, such as `column`, `chunk`, etc. I'll make an issue and link it below to consider the naming of these methods.

Reply:
Added an issue to track this: #37740